src/sentry/static/sentry/app/components/confirm.jsx

```diff
@@ -15,6 +15,7 @@ class Confirm extends React.PureComponent {
      * If true, will skip the confirmation modal and call `onConfirm`
      */
     bypass: PropTypes.bool,
+    title: PropTypes.string,
```

Comment:
Can you make this a separate PR? This looks like it's going to be break a lot of our confirm modals right away.